### DISCUSSION TITLE
concepts/recorder/

### DISCUSSION THREAD

**Discussion by giscus** [go-gost/gost#384] [General]
# concepts/recorder/

https://latest.gost.run/concepts/recorder/

**Comment by pig0224** [comment]
能否支持处理器的数据记录，数据记录可以支持访问目标的method（http\https）数据吗

**Reply by ginuerzh** [reply]
已支持处理器级别的记录器
https://gost.run/concepts/recorder/#recorderservicehandler

**Comment by fiamma06** [comment]
Do you have plans to add the ability to manage recorders via API ?

**Comment by mcat1997** [comment]
能支持handler是chain的service，记录数据吗
